.github/workflows/mongoose-publish.yaml

```diff
@@ -0,0 +1,32 @@
+name: Publish Package
+
+on:
+  push:
+    tags:
+      - "@cerbos/orm-mongoose@v*"
+
+defaults:
+  run:
+    working-directory: mongoose
+
+permissions:
+  id-token: write # Required for OIDC
+  contents: read
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: actions/setup-node@v4
+        with:
+          node-version: "20"
+          registry-url: "https://registry.npmjs.org"
```

Comment:
This is, annoyingly, wrong in the docs: https://github.com/npm/documentation/pull/1748

```suggestion
```